### DISCUSSION TITLE
How to adjust macOS traffic light position?

### DISCUSSION THREAD

**Discussion by MisakaTAT** [wailsapp/wails#3814] [Q&A] [answered]
Just like electron.

```js
const { BrowserWindow } = require('electron')
const win = new BrowserWindow({
  titleBarStyle: 'hidden',
  trafficLightPosition: { x: 10, y: 10 }
})
```

**Comment by leaanthony** [comment]
You can't.

**Reply by MisakaTAT** [reply]
Ok, thanks for your answer. May I ask if there are any plans to support this API in the future?

**Reply by leaanthony** [reply, marked as answer]
No. Cocoa applications conform to Apple's Human Interface Guidelines. The reason Electron allows this, is because they use a fake titlebar.
